### PR TITLE
fix(infra): better search result

### PR DESCRIPTION
**Pull request by EYHN**

### CODE REVIEW

**Comment by EYHN** [comment]
* **#7774** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7774?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by pengx17 — APPROVED** [review]

File: packages/frontend/core/src/modules/docs-search/worker/types.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export type WorkerInput =
     }
   | {
       type: 'doc';
-      docId: string;
+      storageDocId: string;
```
Comment (pengx17) [inline packages/frontend/core/src/modules/docs-search/worker/types.ts:37]: can you explain changes to `storageDocId`? to increase worker's inefficiency?

Comment (EYHN) [reply packages/frontend/core/src/modules/docs-search/worker/types.ts:37]: This is due to historical reasons. The id of the doc stored may not be consistent with the docid. The current solution is to traverse the space map in the rootDoc ​ and query the docid from the storageDocId in the worker.

Comment (EYHN) [reply packages/frontend/core/src/modules/docs-search/worker/types.ts:37]: This is to fix the issue that some old docs not automatically update the index

Comment (pengx17) [reply packages/frontend/core/src/modules/docs-search/worker/types.ts:37]: I see, thanks!

**Pull request merged**